core/src/main/scala/org/apache/spark/storage/BlockManager.scala

```diff
@@ -1074,7 +1074,7 @@ private[spark] class BlockManager(
    * Remove all blocks belonging to the given broadcast.
    */
   def removeBroadcast(broadcastId: Long, tellMaster: Boolean): Int = {
-    logInfo(s"Removing broadcast $broadcastId")
```

Comment:
It would be good to make sure that when the user explicitly `destroy()`s a broadcast, we do have some INFO level logging that we are doing it. I think that can be achieved by dropping this one to debug and logging things higher up in the call stack in the destroy method of either broadcast implementation.